apiserver/src/main/java/org/dependencytrack/resources/v1/BomResource.java

```diff
@@ -134,7 +135,7 @@ public class BomResource extends AbstractApiResource {
                     content = @Content(schema = @Schema(implementation = ProblemDetails.class), mediaType = ProblemDetails.MEDIA_TYPE_JSON)),
             @ApiResponse(responseCode = "404", description = "The project could not be found")
     })
-    @PermissionRequired(Permissions.Constants.VIEW_PORTFOLIO)
+    @DisableAuthorization
```

Comment:
Unfortunately for the mid-term we'll need to continue supporting `@PermissionRequired`. This is relevant for deployments that do not use portfolio ACL, where removing `@PermissionRequired` effectively removes AuthZ entirely.

What this means is that Alpine's `AuthorizationFilter` will need to be aware of the new annotation, and omit AuthZ checks when it's present. If the filter needs to access API server code, consider moving the filter to the `apiserver` module.